### DISCUSSION TITLE
PROXY protocol per TCP listener

### DISCUSSION THREAD

**Discussion by carlhoerberg** [rabbitmq/rabbitmq-server#2870] [Ideas] [answered]
Just throwing this out there. It would be very flexible if one could set the PROXY protocol on individual TCP listeners: 

```
listeners.tcp.default = 0.0.0.0:5672
listeners.tcp.other = 127.0.0.1:5673
listeners.tcp.other.proxy_protocol = true
```

Currently it's all or nothing.

**Comment by yonisim** [comment]
@carlhoerberg have you found a way to achieve this?

**Reply by michaelklishin** [reply, marked as answer]
Proxy Protocol can only be [enabled globally](https://www.rabbitmq.com/docs/networking#proxy-protocol), not per listener.
